### PR TITLE
Introduce code of conduct

### PR DESCRIPTION
**Pull request by mrlegowatch**
Based on contributor-covenant.org version 1.4.

The only edited part is under enforcement the e-mail address:

> Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at sascha.zelzer@gmail.com.

### CODE REVIEW

File: CodeOfConduct.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
```
Comment (jeffdiclemente) [inline CodeOfConduct.md]: A link to this document from README.md or CONTRIBUTING.md would be useful for readers.

**Comment by saschazelzer** [comment]
Actually, I would prefer to use `info@cppmicroservices.org` as contact address (or something similar). If you agree, it would be great to have at least two of you guys (in addition to myself) in the recipient list for this address.

**Comment by jeffdiclemente** [comment]
I am on board with having myself on the ```info@cppmicroservices.org``` recipient list.

**Comment by karthikreddy09** [comment]
Me too …

**Comment by saschazelzer** [comment]
The `info@cppmicroservices.org` address is now operational and be used in the code of conduct document.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -133,3 +133,18 @@ Use your real name (sorry, no pseudonyms or anonymous contributions).
...
+CppMicroServices.org welcomes developers with different backgrounds and a broad range of 
+experience. A diverse and inclusive community will create more great ideas, provide more unique 
+perspectives, and produce more outstanding code. Our aim is to make the CppMicroServices community 
+welcoming to everyone.
+
+To provide clarity of what is expected of our members, CppMicroServices has adopted the code 
+of conduct defined by [contributor-covenant.org][contributor-covenant.org]. This document is used 
+across many open  source communities, and we believe it articulates our values well. 
+
+The full text is on [Code of Conduct](./CodeOfConduct.md).
```
Comment (jeffdiclemente) [inline CONTRIBUTING.md]: This sentence sounds awkward to me.
Some alternatives I can think of...

- The full text can be found within our [Code of Conduct](./CodeOfConduct.md).
- The full text is within our [Code of Conduct](./CodeOfConduct.md).
- Please refer to the [Code of Conduct](./CodeOfConduct.md) for further details.

File: README.md [hunk 3]
```diff
@@ -61,6 +61,20 @@ directory for detailed copyright information.
...
+CppMicroServices.org welcomes developers with different backgrounds and a broad range of 
+experience. A diverse and inclusive community will create more great ideas, provide more unique 
+perspectives, and produce more outstanding code. Our aim is to make the CppMicroServices community 
+welcoming to everyone.
+
+To provide clarity of what is expected of our members, CppMicroServices has adopted the code 
+of conduct defined by [contributor-covenant.org][contributor-covenant.org]. This document is used
+across many open source communities, and we believe it articulates our values well. 
+
+The full text is on [Code of Conduct](./CodeOfConduct.md).
```
Comment (jeffdiclemente) [inline README.md]: This sentence sounds awkward to me. See suggestions from my previous comment regarding this.

**Comment by karthikreddy09** [comment]
+1 for merge

**Pull request merged**